docs/source/pillars/supporting.md

```diff
@@ -0,0 +1,217 @@
+  - This could include having a cost recovery process in place, or setting up a long-term funding mechanism to support projects with TREs. At any given time, you should have funds free to cover all potential foreseen TRE provision for at least 12 months
+  - Recommended
+* - You should minimise the cost of your TRE infrastructure wherever possible
+  - You should have regular reviews of your TRE provision and actively work to bring down costs, streamline provision, and optimise support
+  - Optional
+```
+
+## Procurement
+
+_What the organisation does to ensure the effective sourcing, purchasing and supply of the goods and services that enable the organisation to operate._
+
+```{list-table}
+:header-rows: 1
+:name: tab-procurement
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
+```
+
+## IT Service management
+
+_The implementation and management of quality IT services that meet the needs of the organisation._
+
+```{list-table}
+:header-rows: 1
+:name: tab-it-service-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have a team in place to support projects working with TREs
+  - This may be part of your organisation's IT support team, or separate. Responsibility should be clear and stakeholders should easily be able to access support appropriate to their needs
+  - Mandatory
+```
+
+## Relationship management
+
+_All activities aimed at ensuring a continuous level of engagement is maintained between the organisation and its customers, stakeholders & other interested parties._
+
+### Stakeholder relationships
+
+```{list-table}
+:header-rows: 1
+:name: tab-relationship-stakeholder
+
+* - Statement
+  - Guidance
+  - Importance
+* - You should have a clear process in place for stakeholders to feedback on your TRE infrastructure
+  - This may include a GitHub repository where people can open issues and discussions, communication streams like Slack or email, or forms stakeholders can fill in
+  - Recommended
+```
+
+### Patient and Public Engagement
+
+_What the organisation does to ensure it establishes and maintains mutually beneficial relationships between the organisation and the public in order to maintain trust._
+
+```{list-table}
+:header-rows: 1
+:name: tab-patient-public-engagement
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
+```
+
+## Legal services
+
+_The ability of the TRE organisation to access suitable and timely legal advice._
+
+<!-- Specific requirements? _e.g._ Article 32 of the GDPR requires organisations to regularly test and evaluate the effectiveness of the technical and organisational measures employed to protect personal data, and penetration testing is an effective way of assessing your technical defences. -->
+
+### Legal advisory
+
+_Ability of the organisation to provide suitable and timely legal advice._
+
+```{list-table}
+:header-rows: 1
+:name: tab-legal-services
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
```

Comment:
```suggestion
* - The TRE organisation should identify areas where legal advice may be needed and plan where this will be sourced from.
  - It is likely that legal advice will be necessary for several issues around the handling of sensitive data.
  - Recommended
```

Reply:
@jemrobinson already covered here: https://github.com/sa-tre/satre-specification/blob/main/docs/source/pillars/information_governance.md#policy-regulation-and-management?

Reply:
I would agree with that distinction, then think the wording needs to change to smth like: 
```suggestion
* - The TRE organisation should have ready access to legal advice
  - It is likely that legal advice will be necessary for several issues around the handling of sensitive data. TRE organisations should have ready access to legal advice, including a way to solicit advice and carry out associated actions
  - Recommended
```

Reply:
Have added smth about contracts in reference to point below